nilmtk/dataset_converters/dataport/download_dataport.py

```diff
@@ -128,6 +128,37 @@ def database_assert(database_table):
             or database_table == 'electricity_egauge_minutes'
             or database_table == 'electricity_egauge_seconds'
             ), "Table not compatible with NILMTK"
+
+def view_database_tables(database_username, database_password, 
+                     database_schema):
+    
+
+    database_host = 'dataport.pecanstreet.org'
+    database_port = '5434'
+    database_name = 'postgres'
+
+    try:
+        conn = db.connect('host=' + database_host + 
+                          ' port=' + database_port + 
+                          ' dbname=' + database_name + 
+                          ' user=' + database_username + 
+                          ' password=' + database_password)
+    except:
+        print('Could not connect to remote database')
+        raise
+
+    
+    #Loading university schemas
+    sql_query = ("SELECT table_name" +
+                 " FROM information_schema.views" +
+                 " WHERE table_schema ='" + database_schema + "'" +
+                 " ORDER BY table_name")
+    database_tables=pd.read_sql(sql_query, conn)['table_name'].tolist()
```

Comment:
ensure pep8 please

Reply:
Sure sir. Should I ensure pep8 after you merge the pull request?